src/pods/canvas/components/table/database-table.module.css

```diff
@@ -1,14 +1,31 @@
-.text {
+:root{
+  --primary-background: #041729;
+  --primary-table: #04293A;
+  --primary-header: #ECB365;
+  --primary-text: #ffffff;
+}
```

Comment:
Let's place this on _./App.css_ 

_./App.css_

```diff
#root {
  margin: 0 auto;
  text-align: center;
+  --primary-background: #041729;
+  --primary-table: #04293A;
+  --primary-header: #ECB365;
+  --primary-text: #ffffff;
}
```

By doing this we have a single place where we can update the whole theme